scripts/checkoutCore.js

```diff
@@ -4,24 +4,33 @@ var fs = require('fs');
 var REPOSITORY_NAME = "git@github.com:feedhenry-raincatcher/raincatcher-core.git";
 var LOCATION = "./core";
 
-console.log("Linking core repository");
+// PARAMETERS
+var PULL_RECENT_CHANGES = !!process.env.SHOULD_PULL_CHANGES
+var USE_SHALLOW_CLONE = !!process.env.USE_SHALLOW_CLONE
+var BRANCH_TO_USE = process.env.BRANCH_TO_USE || 'master';
```

Comment:
We could have https://github.com/yargs/yargs as a devDependency and use cli params here instead of only env vars